<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDocDlg</class>
 <widget class="QDialog" name="SelectDocDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Document</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QRadioButton" name="radioMoveExisting">
     <property name="text">
      <string>Move existing document</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioCreateNew">
     <property name="text">
      <string>Create new document</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="toolOverview">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="linePath"/>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioCopyExisting">
     <property name="text">
      <string>Copy existing document</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
